shell/plugins/steve/actions.js

```diff
@@ -96,6 +96,8 @@ export default {
 
       try {
         if (isAdvancedWorkerStore) {
+          // TODO: RC why could there be no worker?
+          // TODO: RC do we do this setup stuff in multiple places?? (like in subscribe)
```

Comment:
just defensive coding, could maybe be abstracted or cleaned up a bit but essentially this ensures that if we're in this branch of the code and the worker crashed and/or hasn't been created yet due to a race condition that we're at least kicking that process off and waiting for it to be ready before we send requests that'll error out and just not get sent.

Reply:
`config` (updated to `initWorker` inline with basic worker init function) now happens in a common place. createWorker is needed on cluster change (we reset $workers)